Helper/ApiClient.php

```diff
@@ -92,7 +92,12 @@ public function oauthAuthentication($storeId)
   public function prepareOrderData($order) 
   {
     $data['email'] = $order->getCustomerEmail();
-    $data['customer_name'] = $order->getCustomerName();
+	$customer = $order->getCustomerName();
+	if($customer == "Guest"){
+		$customer = $order->getBillingAddress();
```

Comment:
please use a different variable to save the result and name it something like billing_address